### ISSUE TITLE
Terminal freezes after redrawing the screen too often or frequent

### ISSUE THREAD

**Issue by t3chn0m4g3** [cockpit-project/cockpit#13575]
Cockpit version: 212
OS: Debian
Page: Terminal
Browser: Chrome

Terminal becomes unresponsive after an unknown times of redrawing the screen (i.e. viewing logs)

Relevant parts of the system log are also useful:
```
Feb 19 13:37:51 electricalsculpture cockpit-tls[598831]: cockpit-tls: gnutls_handshake failed: A TLS fatal alert has been received.
Feb 19 13:37:51 electricalsculpture cockpit-ws[598842]: Connection to session 452 closed
Feb 19 13:37:51 electricalsculpture realmd[599290]: client gone away: :1.1507
Feb 19 13:37:51 electricalsculpture realmd[599290]: released daemon: :1.1507
Feb 19 13:37:51 electricalsculpture su[756075]: pam_unix(su-l:session): session closed for user root
Feb 19 13:37:51 electricalsculpture sudo[756074]: pam_unix(sudo:session): session closed for user root
Feb 19 13:37:51 electricalsculpture cockpit-tls[598831]: cockpit-tls: gnutls_handshake failed: A TLS fatal alert has been received.
Feb 19 13:37:52 electricalsculpture cockpit-tls[598831]: cockpit-tls: gnutls_handshake failed: A TLS fatal alert has been received.
Feb 19 13:37:52 electricalsculpture cockpit-tls[598831]: cockpit-tls: gnutls_handshake failed: A TLS fatal alert has been received.
Feb 19 13:37:52 electricalsculpture cockpit-tls[598831]: cockpit-tls: gnutls_handshake failed: A TLS fatal alert has been received.
```

Steps to reproduce
1. `cat` a log file with 50.000+ lines
2. Terminal will freeze
3. Pressing the `reset` button does not solve the situation only `reload` of the page / cockpit will

**Comment by t3chn0m4g3** [comment]
After upgrading to 213 I can confirm this also accounts for Cockpit version 213.

**Comment by martinpitt** [comment]
This was fixed last May in commit 7eafa1a1164532204620737fb0dd2604f66aa99c .

**Comment by dbms007** [comment]
I installed cockpit 238 on cent os 8.. but still has the same problem.. it freeze on around 180000 lines... why don't u remove the limit ???? if it is freezed.. it is not a console..
